### ISSUE TITLE
Performance issue calculating sha256 for generic packs

### ISSUE THREAD

**Issue by ChipWolf** [itzg/docker-minecraft-server#1281]
### Describe the problem

We've spotted a few servers taking several minutes to start, seemingly hanging on `[init] Checking Forge version information.`

After enabling debug to take a closer look, it seems the startup is hanging on this command for a significant amount of time `sha256sum -c /data/.generic_pack.sum --status`

Removing the sum file appears to allow the startup to proceed much faster.

**Comment by itzg** [comment]
Ah yes, I could see that being a problem. I need to revisit that section and see if the checksum process is overkill there.

**Comment by itzg** [comment]
The latest image now uses sha1 instead of sha256 which should make it faster, but not dramatically faster. 

Let me know if you're interested in a variable to indicate skipping of generic pack update checks in general.

**Comment by itzg** [comment]
...I can also tag the image if that helps for your verification.

**Comment by ChipWolf** [comment]
> ...I can also tag the image if that helps for your verification.

Assuming it'll be on `java8-openj9`?

**Comment by itzg** [comment]
Yes. I now have it build all of the variants (except the old java8 one) with each push to master.

**Comment by ChipWolf** [comment]
Seems to be relatively faster @itzg 
```
2022-02-02T21:01:49Z [init] Checking Forge version information.
2022-02-02T21:03:08Z [init] Disabling whitelist functionality
```

It could do with some additional logging to see what's actually happening without enabling debug there whenever you have a sec, please. I don't recall it taking very long in the past, even a 1-minute check is surprising, for us a flag to disable the check would be handy as the bootstrapping of a generic pack is quicker than this check.

**Comment by itzg** [comment]
I have now added options to skip/force, along with more logs : https://github.com/itzg/docker-minecraft-server/pull/1333
